src/rgw/rgw_cr_rest.h

```diff
@@ -35,6 +39,7 @@ class RGWReadRESTResourceCR : public RGWSimpleCoroutine {
     if (ret < 0) {
       log_error() << "failed to send http operation: " << op->to_str()
           << " ret=" << ret << std::endl;
+      op->put();
```

Comment:
`op` is a `boost::intrusive_ptr`, so its dtor will already `put()` its ref when it goes out of scope here